src/main/java/com/skypro/employee/EmployeeController.java

```diff
@@ -1,64 +1,41 @@
 package com.skypro.employee;
 
-import exception.AlreadyAddedException;
-import exception.EmployeeNotFoundException;
-import exception.EmployeeStorageIsFullException;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Collection;
 
 @RestController
+@RequestMapping("/employee")
 public class EmployeeController {
-    public final EmployeeService employeeService;
+    private final EmployeeServiceInterfaceImpl service;
 
-    public EmployeeController(EmployeeService employeeService) {
-        this.employeeService = employeeService;
+    public EmployeeController(EmployeeServiceInterfaceImpl service) {
```

Comment:
```suggestion
    public EmployeeController(EmployeeServiceInterface service) {
```
инъекцию надо делать по интерфейсу